src/experimental/safe_html_transform/tests/test_setup.py

```diff
@@ -31,3 +31,13 @@ def test_browserlayer(self):
         from experimental.safe_html_transform.interfaces import IExperimentalSafeHtmlTransformLayer
         from plone.browserlayer import utils
         self.assertIn(IExperimentalSafeHtmlTransformLayer, utils.registered_layers())
+
```

Comment:
Those tests will always pass because assertTrue expects an expression, you are passing two arguments instead. I guess assertTrue just takes the first argument as a string (which is true in boolean logic). I would just remove those tests for now.